crates/biome_js_analyze/src/lint/nursery/use_parse_int_radix.rs

```diff
@@ -0,0 +1,290 @@
+                markup!("Will always return "<Emphasis>"NaN"</Emphasis>),
+            ),
+            State::MissingRadix => (
+                markup!("Missing radix parameter"),
+                markup!("Add a non-fractional number between 2 and 36"),
+            ),
+            State::InvalidRadix => (
+                markup!("Invalid radix parameter"),
+                markup!("Radix must be a non-fractional number between 2 and 36"),
+            ),
+            State::RedundantRadix => (
+                markup!("Redundant radix parameter"),
+                markup!("Avoid static radix values of "<Emphasis>"10"</Emphasis>),
+            ),
+        };
+
+        Some(RuleDiagnostic::new(rule_category!(), node.range(), title).note(note))
+    }
+
+    fn action(ctx: &RuleContext<Self>, state: &Self::State) -> Option<JsRuleAction> {
+        let mut mutation = ctx.root().begin();
+        let argument_list = ctx.query().arguments().ok()?.args();
+
+        let (new_args, message) = match state {
+            State::MissingParameters | State::InvalidRadix => return None,
+            State::RedundantRadix => {
+                let first_argument = argument_list.iter().next()?.ok()?;
+                let args = make::js_call_argument_list([first_argument], None);
+
+                (args, markup! { "Remove the radix" })
+            }
+            State::MissingRadix => {
+                let first_argument = argument_list.iter().next()?.ok()?;
+
+                let ten_literal = AnyJsLiteralExpression::JsNumberLiteralExpression(
+                    make::js_number_literal_expression(make::js_number_literal("10")),
+                );
+                let arg = AnyJsCallArgument::AnyJsExpression(
+                    AnyJsExpression::AnyJsLiteralExpression(ten_literal),
+                );
+
+                let args = make::js_call_argument_list(
+                    [first_argument, arg],
+                    Some(
+                        make::token(T![,])
+                            .with_trailing_trivia([(TriviaPieceKind::Whitespace, " ")]),
+                    ),
+                );
+
+                (args, markup! { "Remove add a radix of 10" })
+            }
+        };
+
+        mutation.replace_node::<JsCallArgumentList>(argument_list, new_args);
+
+        Some(JsRuleAction::new(
+            ctx.metadata().action_category(ctx.category(), ctx.group()),
+            ctx.metadata().applicability(),
+            message.to_owned(),
+            mutation,
+        ))
+    }
+}
+
+pub enum State {
+    MissingParameters,
+    MissingRadix,
+    InvalidRadix,
+    RedundantRadix,
+}
+
+#[derive(Debug, Default, Clone, Serialize, Deserialize, Deserializable, PartialEq, Eq)]
+#[cfg_attr(feature = "schemars", derive(JsonSchema))]
+#[serde(rename_all = "camelCase", deny_unknown_fields, default)]
+pub struct UseParseIntRadixOptions {
+    #[serde(default)]
+    pub behavior: Behavior,
+}
```

Comment:
We usually avoid adding options on the first iteration of new rules to avoid inheriting complexity. We usually wait to add options until users have a use case for it.

Reply:
Ah, makes sense. I just figured I'd make it as close to the ESLint implementation. Should I remove it completely or should I comment it out?

Reply:
Let's remove it for the time being